<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1743422259890' stop='1743422261232'>
    <name>My First Test</name>
    <title>My First Test</title>
    <test-cases>
        <test-case start='1743422259890' status='broken' stop='1743422261082'>
            <name>should display the application name</name>
            <title>should display the application name</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>`cy.visit()` failed trying to load:
                
                https://getlinked-main-dev-frontend.vercel.app/
                
                We attempted to make an http request to this URL but the request failed without a response.
                
                We received this error at the network level:
                
                  > Error: getaddrinfo EAI_AGAIN getlinked-main-dev-frontend.vercel.app
                
                Common situations why this would fail:
                  - you don't have internet access
                  - you forgot to run / boot your web server
                  - your web server isn't accessible
                  - you have weird network configuration settings on your computer
                
                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `My First Test`</message>
                <stack-trace>CypressError: `cy.visit()` failed trying to load:
                
                https://getlinked-main-dev-frontend.vercel.app/
                
                We attempted to make an http request to this URL but the request failed without a response.
                
                We received this error at the network level:
                
                  > Error: getaddrinfo EAI_AGAIN getlinked-main-dev-frontend.vercel.app
                
                Common situations why this would fail:
                  - you don't have internet access
                  - you forgot to run / boot your web server
                  - your web server isn't accessible
                  - you have weird network configuration settings on your computer
                
                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `My First Test`
                    at &lt;unknown> (http://localhost:53052/__cypress/runner/cypress_runner.js:131607:74)
                    at visitFailedByErr (http://localhost:53052/__cypress/runner/cypress_runner.js:131006:12)
                    at &lt;unknown> (http://localhost:53052/__cypress/runner/cypress_runner.js:131606:11)
                    at tryCatcher (http://localhost:53052/__cypress/runner/cypress_runner.js:1830:23)
                    at Promise._settlePromiseFromHandler (http://localhost:53052/__cypress/runner/cypress_runner.js:1542:31)
                    at Promise._settlePromise (http://localhost:53052/__cypress/runner/cypress_runner.js:1599:18)
                    at Promise._settlePromise0 (http://localhost:53052/__cypress/runner/cypress_runner.js:1644:10)
                    at Promise._settlePromises (http://localhost:53052/__cypress/runner/cypress_runner.js:1720:18)
                    at _drainQueueStep (http://localhost:53052/__cypress/runner/cypress_runner.js:2430:12)
                    at _drainQueue (http://localhost:53052/__cypress/runner/cypress_runner.js:2423:9)
                    at Async._drainQueues (http://localhost:53052/__cypress/runner/cypress_runner.js:2439:5)
                    at Async.drainQueues (http://localhost:53052/__cypress/runner/cypress_runner.js:2309:14)
                From Your Spec Code:
                    at Context.eval (webpack://cypress_v13_project/./cypress/e2e/my-first-test/getlinked.cy.js:4:11)
                
                From Node.js Internals:
                  Error: getaddrinfo EAI_AGAIN getlinked-main-dev-frontend.vercel.app
                      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>